blogs/rtt/round_trip_time.md

```diff
@@ -0,0 +1,109 @@
+# Network Observability TCP Handshake Round Trip Time Developer Preview
+
+![logo](./images/rtt_logo.png)
+
+By: Dushyant Behl and Julien Pinsonneau
+
+In OCP ensuring efficient packet delivery is crucial for maintaining smooth
+communication between applications. However, due to various factors such
+as network congestion, misconfigured systems, or hardware limitations,
+connections may become slow, impacting overall performance.
+Round Trip Time (RTT) is the duration, measured in milliseconds, from when
+a client sends a request to when it receives a response from a server.
+Mesuring RTT is important for monitoring network health and diagnosing issues.
+
+## RTT calculation using eBPF
+
+// TODO: Explain how RTT is captured and calculed using time diffs
+// between initial SYN & ACK packets using flow_monitor hookpoint
+// Maybe also explain which are the limitations ?
+
+## Potential Use Cases
+
+Flow RTT capture from eBPF `flow_monitor` hookpoint can serve various purposes:
+
+- Network Monitoring: Gain insights into TCP handshakes, helping
+  network administrators identify unusual patterns, potential bottlenecks, or
+  performance issues.
+
+- Troubleshooting: Debug TCP-related issues by tracking latency and identifying
+  misconfigurations.
```

Comment:
Indicate any impact on resources, particularly CPU and memory.
Indicate any security or possible conflict with other apps.